src/Symfony/Bundle/FrameworkBundle/Tests/Functional/ContainerDebugCommandTest.php

```diff
@@ -140,13 +154,15 @@ public function testDescribeEnvVars()
 Symfony Container Environment Variables
 =======================================
 
- --------- ----------------- ------------%w
-  Name      Default value     Real value%w
- --------- ----------------- ------------%w
-  JSON      "[1, "2.5", 3]"   n/a%w
-  REAL      n/a               "value"%w
-  UNKNOWN   n/a               n/a%w
- --------- ----------------- ------------%w
+ ----------------------------- ----------------- ---------------------- -------------%w
+  Name                          Default value     Real value             Usage count%w
+ ----------------------------- ----------------- ---------------------- -------------%w
+  APP_EXAMPLE_ENV_TO_OVERRIDE   n/a               "example_overridden"   0%w
+  APP_EXAMPLE_ENV               n/a               "example_value"        0%w
+  JSON                          "[1, "2.5", 3]"   n/a                    1%w
```

Comment:
I see 2 usages of `JSON` in the config. https://github.com/symfony/symfony/pull/49161/files#diff-9aad08d07b3694cbc7c5e0edf48e28ee4f47122f7ae6559e337c3df7e2ea7b2eR34-R35